src/panels/lovelace/editor/card-editor/hui-dialog-delete-card.ts

```diff
@@ -32,6 +32,7 @@ export class HuiDialogDeleteCard extends LitElement {
       this._cardConfig = deepFreeze(this._cardConfig);
     }
     await this.updateComplete;
+    fireEvent(this, "iron-resize");
```

Comment:
firing the event from `this` will never reach the dialog as that is lower in the dom tree and the event will go up, you would need to fire it from the dialog element. 

You don't get problems with resizing the dialog now? With large cards?

Reply:
I didn't hit any issues when firing it from a larger card. Had like 20 entities and resized fine. Updated to query for the dialog and use that